Sources/PulseUI/Features/Console/ConsoleView.swift

```diff
@@ -12,9 +12,9 @@ extension ConsoleView {
     ///
     /// - parameters:
     ///   - store: The store to display. By default, `LoggerStore/shared`.
-    ///   - mode: The console mode. By default, ``ConsoleMode/all``. If you change
-    ///   the mode to ``ConsoleMode/network``, the console will only display the
-    ///   network messages.
+    ///   - mode: The initial console mode. By default, ``ConsoleMode/all``. If you change
+    ///   the mode to ``ConsoleMode/network``, the console will display the
+    ///   network messages up on appearance.
```

Comment:
Correct me if I misunderstood, but the original documentation made me think I could completely hide the `All` and `Logs` tab buttons. Looking at the code and final outcome it turned out to only configure the initial mode. Hence why I updated the documentation.

Reply:
I'm not sure it works on macOS, but it does work according to the documentation on other platforms.

![Uploading Screenshot 2024-07-01 at 8.15.08 AM.png…]()